editoast/src/views/timetable.rs

```diff
@@ -287,7 +287,7 @@ async fn conflicts(
     }
 
     let db_pool = app_state.db_pool_v2.clone();
-    let redis_client = app_state.redis.clone();
+    let valkey_client: std::sync::Arc<crate::ValkeyClient> = app_state.valkey.clone();
```

Comment:
Why do you need the type to be explicit now? Can't you keep it like it was before?